### PR TITLE
Update Dartdoc for SceneBuilder#addPlatformView

### PR DESCRIPTION
**Pull request by amirh**

### CODE REVIEW

File: lib/ui/compositing.dart [hunk 1]
```diff
@@ -274,7 +274,20 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
...
+  /// On iOS this layer splits the current output surface into 2 surfaces, one for the scene nodes
+  /// preceding the platform view, and one for the scene nodes following the platform view.
+  ///
+  /// ## Performance impact
+  ///
+  /// Adding an additional surface doubles the amount of graphics memory directly used by Flutter
+  /// for output buffers. Quartz might allocated extra buffers for compositing the Flutter surfaces
+  /// and the platform view.
+  /// 
+  /// With a platform view in the scene, Quarts has to composite the 2 Flutter surfaces and the
```
Comment (cbracken) [inline lib/ui/compositing.dart]: s/Quarts/Quartz/

Comment (cbracken) [inline lib/ui/compositing.dart]: uber-nit: s/2/two/

Comment (amirh) [reply lib/ui/compositing.dart]: done

Comment (amirh) [reply lib/ui/compositing.dart]: done

**Comment by cbracken** [comment]
lgtm

**Pull request merged**